### PR TITLE
Node consistency 81 90

### PR DESCRIPTION
**Pull request by martinstacey**
### Purpose
The following PR addresses nodes' names and descriptions to make them more descriptive for beginners.

**81) NODE: List.Deconstruct**
-CHANGES:
Output description:
first: type: var[]..[] (first item in the list)
rest: type: var[]..[] (rest of the list)

**82) NODE: List.Map**
-CHANGES:
Input: fx -> function
Output: mapped -> list
Output description list: type: var[]..[] (mapped list)

**83) NODE: List.Cycle**
-CHANGES:
Output description list: type: var[]..[] (list of repeated lists)

**84) NODE: Sequence**
-CHANGES:
Input description: Amount of numbers or letters in the sequence
Output seq to list
Output description list: type: var[]..[] (New list)

**85) NODE: Range**
-CHANGES:
Output seq to list
Output description list: type: var[]..[] (New list)

**86) NODE: TimeSpan.Components**
-CHANGES:
Output description:
days: type: int (amount of days)
hours: type: int (amount of hours)
minutes: type: int (amount of minutes)
seconds: type: int (amount of seconds)
milliseconds: type: int (amount of milliseconds)

**87) NODE: Object.IsNull**
-CHANGES:
Description: Determines if the given object is null.

**88) NODE: Object.Identity**
-CHANGES:
Input: obj to object
Output : var[]..[] to object
Output object description: Same object

**89) NODE: DateTime.TimeOfDay**
-CHANGES:
Output: var to TimeSpan
Output Description: TimeSpan since midnight

**90) NODE: DateTime.Date**
-CHANGES:
Output: var to date
Output Description: date from DateTime

image

### Reviewers
@SHKnudsen
@Amoursol 
@QilongTang 
@aparajit-pratap 

### FYIs
-No FYI

### CODE REVIEW

**Comment by Amoursol** [comment]
@martinstacey Are the outputs descriptions in this format: `type: var[]..[] (New list)` ? If so, please inverse it to `New list of type: var[]..[]`

**Review by Amoursol — APPROVED** [review]

File: src/DynamoCore/Properties/Resources.Designer.cs [hunk 1]
```diff
@@ -1301,7 +1301,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to New sequence.
+        ///   Looks up a localized string similar to type: var[]..[] (New list).
```
Comment (aparajit-pratap) [inline src/DynamoCore/Properties/Resources.Designer.cs:1304]: The format we're following is that the type is in brackets like @Amoursol notes below.

File: src/Libraries/CoreNodes/List.cs [hunk 2]
```diff
@@ -423,8 +423,8 @@ public static object MaximumItem(IEnumerable<object> list)
         ///     except the first.
         /// </summary>
         /// <param name="list">List to be split.</param>
-        /// <returns name="first">First item in the list.</returns>
-        /// <returns name="rest">Rest of the list.</returns>
+        /// <returns name="first">type: var[]..[] (first item in the list)</returns>
```
Comment (aparajit-pratap) [inline src/Libraries/CoreNodes/List.cs:426]: As above descriptions must come first followed by the type in brackets.

**Review by aparajit-pratap — CHANGES_REQUESTED** [review]
@martinstacey please address review comments before we can merge this PR. Thanks.

File: src/Libraries/CoreNodes/List.cs (hunk 2 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/List.cs]: Addressed in f347775

File: src/DynamoCore/Properties/Resources.Designer.cs (hunk 1 above)
Comment (martinstacey) [reply src/DynamoCore/Properties/Resources.Designer.cs]: Addressed in f347775

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**